.editorconfig

```diff
@@ -123,6 +123,8 @@ dotnet_style_predefined_type_for_locals_parameters_members = true
 
 # Use expression body if a local function is a single line.
 csharp_style_expression_bodied_local_functions = when_on_single_line
+# This rule is buggy and not enforced for builds. ':warning' will at least enforce it in the IDE.
+csharp_style_expression_bodied_lambdas = when_on_single_line:warning
```

Comment:
missing newline. 

Do I understand correctly that make check / GH actions won't detect this violation?

Reply:
Wanted it grouped with the other expression body option, fixed the preceeding comment for that.

You are correct. The desired setup (below) fails to report any violations. The IDE-only `:severity` syntax gives us something for now. But as it is IDE-only, command lines builds (and thus CI) won't get them.
```ini
csharp_style_expression_bodied_lambdas = when_on_single_line
dotnet_diagnostic.IDE0053.severity = warning
```

Reply:
Our style it to have a newline before comments. I assume those don't trigger inside this file

Reply:
Updated.